plugins/fabric8-cdn-support/src/main/java/com/redhat/che/cdn/CdnSupportService.java

```diff
@@ -265,8 +265,8 @@ private PluginMeta getEditorMeta() throws InfrastructureException {
         pluginFQN.getRegistry() == null
             ? pluginRegistry.clone()
             : UriBuilder.fromUri(pluginFQN.getRegistry());
-
-    URI editorURI = uriBuilder.path(pluginFQN.getId()).build();
+    // slash after 'id' is required to avoid '301 permanently moved' http error
+    URI editorURI = uriBuilder.path(pluginFQN.getId() + "/").build();
```

Comment:
The technically correct URL here would be 
```java
uriBuilder.path(pluginFQN.getId()).path("meta.yaml").build()
```
We're getting a `301` because we're querying a directory, not a specific file. I believe it's only returning the meta.yaml on this path because the meta.yaml is typically the only file in that directory (which is not required -- i.e. this could stop working if someone adds a `README.md` to a plugin's directory).

Reply:
good, point - we probably need to fall back on `meta.yaml`

Reply:
fixed